index.html

```diff
@@ -3,10 +3,63 @@
       <head>
         <meta charset="UTF-8">
         <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Poppins">
         <link rel="stylesheet" href="styles/main.css">
-        <title>JT</title>
+        <title>JT's Portfolio</title>
       </head>
 <body>
-    <header> <h1>Jud&aacute; Teixeira</h1> </header>
+  
+  <header>
+    <div>
+      <a href="#" class="logo">
+        <img src="images/banner.png" alt="Jt-logo">
+      </a>
+    </div>
+    
+
+    <div>
+      <nav>
+        <a href="#" class = "menu">
+        <img src="images/menu.png" alt="Menu">
+        </a>
+     </nav>
+    </div>
+  </header>
+  
+  <div class="main">
+  
+  <div class="content">
+
+    <h1>I'm Jud&aacute; <br> Glad to see <br> you!</h1>
+    <p>
+      I'm a brazillian web developer! 
+      I can help you build a product, feature 
+      or website Look through some of my work and experience! 
+      If you like what you see and have a project you need coded, 
+      don't hesitate to contact me.
+      </p>
+
+  </div>
+
+  <div class="footer">
+    <h3>LET'S CONNECT</h3>
+  <ul class="icon-bar">
+    <li><a href="https://twitter.com/87juda">
+      <img src="images/twitter.png" alt="Twitter">
+    </a></li>
+    <li><a href="https://github.com/mrjuda">
+      <img src="images/github.png" alt="Github">
+    </a></li>
+    <li><a href="https://www.linkedin.com/in/judateixeira/">
+      <img src="images/linkedin.png" alt="Linkedin">
```

Comment:
- [ ] (Optional) Another major project requirement states that you should use the styles for your template provided by Figma. I think it would be nice to make your portfolio look exactly like that of **_Figma's template_** that you chose. This way, you would be meeting project requirements.

## Your Design 

![image](https://user-images.githubusercontent.com/77106962/144479260-e9e858ef-ed1e-47cb-9ecf-0b926aaf2201.png)


## Figma's Design

![image](https://user-images.githubusercontent.com/77106962/144479328-cd3b28bf-ccdd-414e-b0bc-0d31bd28aae3.png)

### I strongly recommend that you make your portfolio look like that of the screenshot above. 👍